### PR TITLE
Design pass 

### PR DESCRIPTION
**Pull request by anastasia**
- buttons and links should be more consistent now
- typography should be more consistent with the updated designs
- touched a bit of page structure as well
- updated some language, currently we're inconsistent with the titles — I think they should be upper case for the first letter only, like Accessing data, instead of Accessing Data. What do you think?

### CODE REVIEW

File: capstone/capweb/templates/about.html [hunk 1]
```diff
@@ -283,60 +280,39 @@ <h2 class="subtitle" id="partners">Friends & Partners</h2>
             Harvard Law School Library
           </a>
         </span>
-        <span class="color-red">&bull;</span>
         <span>
-          LIL is part of the Harvard Law School Library.
+           is part of the Harvard Law School Library.
```
Comment (jcushman) [inline capstone/capweb/templates/about.html]: Does this say "Harvard Law School Library is part of the Harvard Law School Library."?

**Review by jcushman — COMMENTED** [review]
Nice! I love the button stuff.

I'm fine with sentence case instead of title case for titles -- I found https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98 which suggests sentence case is a little easier to do consistently and a little less formal.

Just one note about what I think is a typo on the About page, though I could be confused.

File: capstone/capweb/templates/about.html (hunk 1 above)
Comment (anastasia) [reply capstone/capweb/templates/about.html]: Yes it's the law of symmetry we learned about this

Comment (bensteinberg) [reply capstone/capweb/templates/about.html]: As Prince says, there's joy in repetition.

**Comment by ChefAndy** [comment]
It thinks it looks great. Overall it feels much tighter and smoother.

On my pass I might play around with some alternatives to the titles or the top section layout on small screens— some of them get pretty cramped. 

Screen Shot 2019-04-12 at 12 50 30 PM

**Review by ChefAndy — APPROVED** [review]

**Comment by bensteinberg** [comment]
This build is erroring (not failing):
```
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/sass.py", line 709, in compile
    raise CompileError(v)
sass.CompileError: Error: ".list-group-item" failed to @extend ".font-header".
       The selector ".font-header" was not found.
       Use "@extend .font-header !optional" if the extend should be able to fail.
        on line 92 of staticfiles/css/scss/viz.scss
>>   @extend .font-header;
```

**Comment by anastasia** [comment]
@ChefAndy hey if it's OK with you, I'm going to do a first pass at it now in this PR. I don't like the idea of breaking the way things look, and small screen sizes look better in prod now.

**Comment by anastasia** [comment]
Any chance anyone has ideas for why this is failing? I'm running `npm run build` locally and not seeing any build files created. Running this test locally doesn't fail, either, it only seems to fail on travis. Am I forgetting a step?

**Comment by jcushman** [comment]
Andy and I think if you run `npm install` first then your build should match Travis -- Ben updated some packages a couple of weeks ago. If that doesn't work, try deleting `node_modules` and reinstalling.

**Comment by ChefAndy** [comment]
💯
https://www.youtube.com/watch?v=8a-HfNE3EIo

**Pull request merged**